### ISSUE TITLE
Android issue

### ISSUE THREAD

**Issue by Everhusk** [jodybrewster/react-native-linkedin-login#1]
![Image of error](https://i.imgur.com/yI45ik6.png)

Hey great package! I just tried this package on android by following the instructions but first it throws a compile error due to a naming collision with react native. After deleting the react native folder inside the node_modules folder it compiles, but throws the error in the picture above. Any ideas?

**Comment by jodybrewster** [comment]
Sorry didn't realize there was an issue, my bad I took to long.  My initial thoughts are that the android java code has updated at some point and the javascript code didn't.  I changed that init method sometime early on in the project.  I'd check the javascript code in the module by going to node_modules/react-native-linkedin-login/example in the root of your project.  Just make sure the javascript code in the example is close to your code. They've gotten out of sync.  

In general I've migrated the JS code to ES6 and support for the significant dependencies change to RN 0.18.
